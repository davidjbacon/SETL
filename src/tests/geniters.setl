PROGRAM p;
  x := 0;
  PRINT({x: DOING x := f(x); WHILE x < 100});
  PRINT(x);
  PRINT({x: x IN [1..10]});
  PRINT(STR x);
  PROC f(x);
    RETURN(1+2*x);
  END;
END;
