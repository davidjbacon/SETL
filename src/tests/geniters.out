{1 3 7 15 31 63}
127
{1 2 3 4 5 6 7 8 9 10}
*
